frontend/src/metabase/parameters/components/widgets/ParameterFieldWidget.jsx

```diff
@@ -64,6 +32,19 @@ export default class ParameterFieldWidget extends Component<*, Props, State> {
     };
   }
 
+  static propTypes = {
```

Comment:
I thought the deal was not to use `static` anymore.
Can we please extract propTypes and apply them in the same manner like we'd do for the functional component?

Reply:
Yes sure, done.